app/transports/api/servers/api.go

```diff
@@ -60,6 +60,7 @@ func Build() fx.Option {
 
 			rtr.Get("/{address}", s.get)
 			rtr.Get("/", s.list)
+			rtr.Get("/full", s.fulllist)
```

Comment:
don't we want to apply the cache to the main /servers endpoint too? that's the one I think most people use

Reply:
main/old ones use cached files as well